include/config/config_movement.h

```diff
@@ -103,6 +103,11 @@
  */
 #define HANGING_FIX
 
+/**
+ * Buffers an A input if you jump off a slope during the landing lag
+ */
+#define SLOPE_FIX
```

Comment:
HackerSM64 already has a built-in "slope fix" that does something else, so this should probably be called something else (and disabled by default).

Reply:
This change (at least in binary hacks) is usually called "slope fix", though there's a bunch of things labeled "slope fix" so it's pretty vague. I renamed the define to "SLOPE_BUFFER" to be more descriptive, but I'm willing to change it if people think there's a better name than that.

(I also moved it down to the other slope define, as I think organizing the slope defines together is probably a good idea)